### PR TITLE
Add Radio

### PR DESCRIPTION
**Pull request by alexkdeveloper**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.alexkdeveloper.radio.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "app-id": "com.github.alexkdeveloper.radio",
```
Comment (hfiguiere) [inline com.github.alexkdeveloper.radio.json:2]: It should be:

```suggestion
  "app-id": "io.github.alexkdeveloper.radio",
```

File: com.github.alexkdeveloper.radio.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+  "app-id": "io.github.alexkdeveloper.radio",
```
Comment (hfiguiere) [inline com.github.alexkdeveloper.radio.json:2]: the manifest filename no longer match the app-id.

also remember only the dbus name matching the app-id is granted ownership by default. And I suspect that the icons and desktop files will also be mis-named in that situation.

**Comment by hfiguiere** [comment]
bot, build io.github.alexkdeveloper.radio

**Comment by alexkdeveloper** [comment]
bot, build io.github.alexkdeveloper.radio

**Comment by alexkdeveloper** [comment]
bot, build io.github.alexkdeveloper.radio

**Comment by alexkdeveloper** [comment]
bot, build io.github.alexkdeveloper.radio

**Comment by alexkdeveloper** [comment]
bot, build io.github.alexkdeveloper.radio

**Comment by alexkdeveloper** [comment]
When can I expect the app to be published?

**Pull request closed without merging**